packages/frontend/src/components/kpi-simulation-chart/index.tsx

```diff
@@ -253,9 +257,11 @@ export function KpiSimulationChart({
             const yearMultiplier = SECONDS_IN_YEAR / campaignDurationSeconds;
             const aprPercentage = rewardsRatio * yearMultiplier * 100;
 
-            if (maxAprPercentage < aprPercentage)
+            aprPercentages.push(aprPercentage);
```

Comment:
Doing this means we still show the APR % above the capped max in the tooltip, but not in the chart, not sure if this is fine or not.

Reply:
Yeah this is actually wanted. The problem is mainly with the chart, but always showing the APR is more than fine IMHO.